Skype/UCWA/message_ref.md

```diff
@@ -71,31 +72,32 @@ The user must have at least one of these scopes for operations on the resource t
 |message|High|conversation|Delivered when a message is started for an incoming instant message.|
```

Comment:
This might not fix the issue, but let's move the first pipe to the beginning of line 71.